lib/advanced-open-file-view.coffee

```diff
@@ -201,12 +217,19 @@ class AdvancedFileView extends View
       + ' icon-file-directory-create'\
       + ' icon-alert'
     if icon? then @message.addClass 'icon icon-' + icon
-    @message.text str or "Enter the path for the file/directory. Directories end with a '" + path.sep + "'."
+    @message.text str or """
```

Comment:
It was in the CS example for multiline strings, but apparently it's not necessary. Yay!